certbot/certbot/_internal/log.py

```diff
@@ -11,6 +11,10 @@
 Special care is taken by both methods to ensure all errors are logged
 and properly flushed before program exit.
 
+The default verbosity for the full terminal logger is INFO. The preferred
```

Comment:
Can we add a line that makes it clear what this is for, in contrast to the `certbot.display` modules? Maybe something like "This module is for operational messages that give information about what Certbot is doing under the hood." (Don't love that wording, feel free to improve.)

Reply:
OK, I've tried to do that.